### PR TITLE
[IMP] mail_tracking: Adapt test to set (not order preserved)

### PR DESCRIPTION
**Pull request by Tardo**
This PR fix the test

Cc @tecnativa

### CODE REVIEW

File: mail_tracking/tests/test_mail_tracking.py [hunk 1]
```diff
@@ -137,8 +137,10 @@ def test_email_cc(self):
         self.assertFalse(email_cc[1])
         # suggested recipients
         recipients = self.recipient.message_get_suggested_recipients()
-        self.assertEqual(recipients[self.recipient.id][0][1],
-                         'unnamed@test.com')
+        suggested_mails = {
+            email[1] for email in recipients[self.recipient.id]
+        }
+        self.assertTrue('unnamed@test.com' in suggested_mails)
```
Comment (pedrobaeza) [inline mail_tracking/tests/test_mail_tracking.py]: use `assertIn` instead

Comment (Tardo) [reply mail_tracking/tests/test_mail_tracking.py]: Done, thanks!

**Review by pedrobaeza — APPROVED** [review]
Fast-tracking for fixing Travis (it gets a random error)

**Pull request closed without merging**